### ISSUE TITLE
Problem with mv-initial-items > 1: Expressions are not evaluated after the first item

### ISSUE THREAD

**Issue by LeaVerou** [mavoweb/mavo#826]
Testcase: https://codepen.io/leaverou/pen/BamZpKG?editors=1100

@DmitrySharabin is this a regression?

**Comment by DmitrySharabin** [comment]
I've noticed this thing quite a long time ago. But as usual, I haven't reported it since I didn't expect `mv-initial-items` should work with values other than `0`. 🤦‍♂️


> is this a regression?
I'll bisect to get to know that.

**Comment by DmitrySharabin** [comment]
Yup, it's a regression. The first bad commit: 2e119ce2f9b0267cea46bcc527db832d38bae90e

**Comment by LeaVerou** [comment]
Let's defer this, since as you point out, `mv-initial-items` > 1 is rare.

**Comment by DmitrySharabin** [comment]
It seems we face this issue not only when `mv-initial-items` > 1: **none** of the expressions are evaluated _after_ the first item.

Testcase: https://codepen.io/dmitrysharabin/pen/BaeEVme
